engine/table/src/main/java/io/deephaven/engine/table/impl/select/MatchFilter.java

```diff
@@ -681,12 +717,16 @@ public boolean canMemoize() {
     public WhereFilter copy() {
         final MatchFilter copy;
         if (strValues != null) {
+            final WhereFilter failover = failoverFilter != null ? failoverFilter.getIfCached() : null;
+            if (failover != null) {
+                return failover.copy();
+            }
+
```

Comment:
This is suspicious. We might resolve different query scope params on a new init.
```suggestion
```